core/dslmcode/shared/drupal-7.x/modules/elmsln_contrib/cis_connector/modules/features/cis_section/cis_section.views_default.inc

```diff
@@ -153,14 +161,6 @@ function cis_section_views_default_views() {
   $handler->display->display_options['fields']['edit_node']['field'] = 'edit_node';
```

Comment:
Will do - I need to double check that this was not the one we had exported in courses / mooc already.